src/3rd_party/CMakeLists.txt

```diff
@@ -241,7 +243,6 @@ endif()
 
 add_custom_target(install-3rd_party
   DEPENDS ${install-3rd_party_logger_var}
-  DEPENDS libbson
   DEPENDS Boost
```

Comment:
@BSolonenko  boost still does not use namespaces

Reply:
@LuxoftAKutsan this will be fixed when I will create PR with refactoring CMakeLists for 3rd-party libraries.